src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -5,9 +5,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionService {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     MovieSession get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
```

Comment:
Consider changing the return type of the `update` method to `MovieSession` or `void` to better reflect its operation. Returning a Boolean might not provide enough information about the update process.